Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Github9536.xaml.cs

```diff
@@ -0,0 +1,23 @@
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+using Xamarin.Forms.Xaml;
+
+namespace Xamarin.Forms.Controls.Issues
+{
+#if APP
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+#endif
+    [Preserve(AllMembers = true)]
+    [Issue(IssueTracker.Github, 9536, "PopModalAsync hides navigation bar on macOS", PlatformAffected.macOS)]
+    public partial class Github9536:ContentPage
+    {
+        public Github9536()
+        {
+            InitializeComponent();
+        }
+        
+        void Button_Clicked(object sender, System.EventArgs e) {
+            Navigation.PushModalAsync(new NavigationPage(new Issue2964.ModalPage()));
```

Comment:
I used an existing modal page here. I hope this is okay. Not a fan of code repetition, but could understand if necessary.